zmon_agent/discovery/kubernetes/kube.py

```diff
@@ -62,3 +62,9 @@ def get_endpoints(self, namespace=DEFAULT_NAMESPACE) -> pykube.query.Query:
 
     def get_ingresses(self, namespace=DEFAULT_NAMESPACE) -> pykube.query.Query:
         return pykube.Ingress.objects(self.client).filter(namespace=namespace)
+
+    def get_persistentvolumes(self, namespace=DEFAULT_NAMESPACE) -> pykube.query.Query:
```

Comment:
In theory, namespace is not used here, but the `get_all()` call at https://github.com/avaczi/zmon-agent-core/blob/cebf24ea0ba0cb2b970b43c99a1581b1734c2d4c/zmon_agent/discovery/kubernetes/cluster.py#L134 assumes it's there